tsconfig.json

```diff
@@ -10,13 +10,13 @@
     // "allowJs": true,                       /* Allow javascript files to be compiled. */
     // "checkJs": true,                       /* Report errors in .js files. */
     // "jsx": "react",                     /* Specify JSX code generation: 'preserve', 'react-native', or 'react'. */
-    // "declaration": true,                   /* Generates corresponding '.d.ts' file. */
+    "declaration": true /* Generates corresponding '.d.ts' file. */,
```

Comment:
There's no issue with the mock keplr type check with this?

Reply:
Not on my local. Are you having any issues?

Reply:
No